microprofile-jwt/src/main/java/org/jboss/eap/qe/microprofile/jwt/tools/JsonWebToken.java

```diff
@@ -0,0 +1,113 @@
+package org.jboss.eap.qe.microprofile.jwt.tools;
+
+import java.io.UnsupportedEncodingException;
+import java.security.GeneralSecurityException;
+import java.security.PrivateKey;
+import java.security.Signature;
+import java.util.Base64;
+
+/**
+ * A class representing JsonWebToken
+ */
+public class JsonWebToken {
```

Comment:
Why do we need this file?

Isn't available any standard class for this ?

How will customers do this stuff ? If each of them will do the same WF/EAP should provide it for them.

Reply:
https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/38#discussion_r350114220